web/src/main/webapp/app/app.js

```diff
@@ -421,7 +445,11 @@
     // Get a list of all modules to find all files to load.
     getUser().then(function(data) {
       user = data;
-
+      if (user && user.parameters && user.parameters.launchId !== undefined) {
+        if (sessionStorage.getItem('launchId') === null) {
```

Comment:
Should '== null' be removed and that you just leave !sessionStorage.getItem('launchId') If it is false it will be undefined, null or empty string.